src/config/env.js

```diff
@@ -8,7 +8,7 @@ const USE_EEG = import.meta.env.VITE_USE_EEG === "true" && USE_ELECTRON; // Whet
 const USE_PHOTODIODE = import.meta.env.VITE_USE_PHOTODIODE === "true" && USE_ELECTRON; // whether or not the photodiode is in use
 
 // Configuration object for Honeycomb
-const config = {
+const env = {
```

Comment:
You can consolidate this a little if you'd like! 
```suggestion
export default {
```